app/assets/javascripts/live_search.js

```diff
@@ -109,15 +109,29 @@
     }
   };
 
+  LiveSearch.prototype.getAdditionalCustomDimensionsForPageView = function() {
```

Comment:
Is it possible to use something like this: https://github.com/alphagov/prototype-performance-platform-alpha/blob/385a021a526b82caf70c75d703391a584c9ba718/govuk_modules/govuk_frontend_toolkit/docs/analytics.md#set-custom-dimensions-before-tracking-pageviews